charts/opentelemetry-operator/UPGRADING.md

```diff
@@ -1,5 +1,28 @@
 # Upgrade guidelines
 
+## 0.56.0 to 0.57.0
```

Comment:
We need to add a note here and in the readme that if you were using `--skip-crds` that you now need to set `crds.create=false`

Reply:
Done.

Reply:
Can you add a note to the README as well that this chart does not support `--skip-crds` and instead needs `crds.create=false`,